tests/transports/test_all_plugins.py

```diff
@@ -1209,25 +1055,19 @@ def test_transfer_big_stdout(custom_transport):
     line_repetitions = min_file_size_bytes // len(file_line_binary) + 1
     fcontent = (file_line_binary * line_repetitions).decode('utf8')
 
-    with custom_transport as trans:
+    with custom_transport as transport:
+        transport: Transport
```

Comment:
This line is a bit out of blue (?) is not practiced in the similar places..
```suggestion
```

Reply:
Right, I was putting it here only for letting the LSP help me to autocomplate on methods. Removed.